tests/test_extension_amber.py

```diff
@@ -28,6 +28,12 @@ def test_list_in_dict(snapshot):
     assert {"value": lines} == snapshot
 
 
+def test_deeply_nested_list_in_dict(snapshot):
```

Comment:
nb. technically not a list?
```suggestion
def test_deeply_nested_multi_line_in_dict(snapshot):
```